ocs_ci/ocs/ui/page_objects/encryption_module.py

```
@@ -92,6 +88,68 @@ def _get_encryption_summary(self, context_key):
         return encryption_summary
 
+    @staticmethod
+    def _encryption_header_text(header_element) -> str:
+        """Return visible header label (PF6 h6 elements may have empty .text)."""
+        text = (header_element.text or "").strip()
+        if text:
+            return text
+        return (header_element.get_attribute("textContent") or "").strip()
+
+    @staticmethod
+    def _encryption_row_status_enabled(header_element):
```

Ruff (0.15.20)

[warning] Missing return type annotation for staticmethod `_encryption_row_status_enabled`

Add return type annotation: `bool`

(ANN205)